src/test/LiquidityMathTest.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {LiquidityMath} from "../libraries/LiquidityMath.sol";
+
+contract LiquidityMathTest {
```

Comment:
I feel like we dont need this file right? Can't `LiquidityMath.t.sol` just call the library directly?
I know some of our other test files are like that too, I want to clean them up

Reply:
I can move this to `LiquidityMath.t.sol`. But it's required to test positive and negative cases using [try/catch](https://docs.soliditylang.org/en/latest/control-structures.html#try-catch).

> The try keyword has to be followed by an expression representing an external function call or a contract creation (new ContractName()).

Reply:
ohhh very interesting, i like this being a contract in its own file then if its needed for try catch! Sorry for the back and forth 😅